src/Control/Effect/Parse.hs

```diff
@@ -27,3 +34,24 @@ parse :: (Member Parse sig, Carrier sig m)
       -> Blob
       -> m term
 parse parser blob = send (Parse parser blob pure)
+
+
+parseWith
+  :: (Carrier sig m, Member (Error SomeException) sig, Member Parse sig)
```

Comment:
Can we put the `::` on the same line? Doing so enables me to grep for `parseWith ::` and know that I’m going to jump to (probably) the correct definition.

I would also love to have this yield `Error NoLanguageForBlob` rather than `SomeException`, but we don’t necessarily have to fix that part right now.

Reply:
This is intended to yield stable diffs under renames. I’d really rather we used this style given how much re-indentation we tend to waste time doing and reading.